test/apiv2/test-apiv2

```diff
@@ -256,7 +256,7 @@ function t() {
         for arg; do
             case "$arg" in
                 # This is just some hack to avoid adding `-d {}` to curl for endpoints where we really need an empty body.
-                # --disable makes curl not lookup the curlrc file, it't should't effect the tests in any way.
+                # --disable makes curl not lookup the curlrc file, it shouldn't effect the tests in any way.
```

Comment:
Close! You fixed one of my nits, but not the other (should be "affects" with an "a"). I won't block over it; I'm sure the grammar checker will catch it on a future sweep.